Functions/xASL_io_ReadDataPar.m

```diff
@@ -82,6 +84,9 @@
 if nargin < 1 || isempty(pathDataPar) || ~exist(pathDataPar, 'file')
```

Comment:
If the first argument is not provided, it gives an error that the data par doesn't exist

Reply:
Sure, easy fix: 4dbefa8 👍